ksql-streams/src/test/java/org/apache/kafka/streams/kstream/NamedTestAccessor.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.apache.kafka.streams.kstream;
+
+public final class NamedTestAccessor {
```

Comment:
Needed in tests to get at the `String` within the `Named` instance.  Bit of a hack. Could use a factory to build `Named` instances, but that was more work...